### PR TITLE
HCIDOCS-550: Update supported firmware version for idrac 15th and 16t…

### PR DESCRIPTION
**Pull request by kdrake20**
**Fixes:** [HCIDOCS-550](https://issues.redhat.com/browse/HCIDOCS-550)

**For:** OCP 4.16, 4.17, 4.18

[**Preview**](https://85307--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/ipi/ipi-install-prerequisites.html#ipi-install-firmware-requirements-for-installing-with-virtual-media_ipi-install-prerequisites)

**Approvals -** `/lgtm`:
- [x] SME
- [x] QE
- [x] Peer Review

### CODE REVIEW

**Comment by kdrake20** [comment]
**Who:**
@elfosardo, SME
@jadhaj , QE

**Summary:** Hello! Could you please review this content? Please provide feedback directly in the PR.
Comment /lgtm if no changes are required. Thank you!

**Next Steps:**
- [ ] Receive /lgtm from SME
- [ ] Receive /lgtm from QE

File: modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc [hunk 1]
```diff
@@ -17,6 +17,7 @@ Red Hat does not test every combination of firmware, hardware, or other third-pa
 [cols="1,1,1",options="header"]
 |====
 | Model | Management | Firmware versions
+| 11th Generation | iLO6 | 1.57 or later
```
Comment (elfosardo) [inline modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc:20]: please move this change to a different PR, it's unrelated, thanks

**Review by elfosardo — CHANGES_REQUESTED** [review]

**Comment by elfosardo** [comment]
/lgtm
thanks!

**Review by opayne1 — COMMENTED** [review]
lgtm!

**Pull request merged**